autogluon/utils/tabular/ml/learner/default_learner.py

```diff
@@ -158,17 +158,33 @@ def general_data_processing(self, X: DataFrame, X_val: DataFrame, holdout_frac:
         else:
             y_val = None
 
+        if self.submission_columns:
+            X = X.drop(self.submission_columns, axis=1, errors='ignore')
```

Comment:
Can we have print statement that these cols will be dropped

Reply:
added